pom.xml

```diff
@@ -229,8 +206,8 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
                     <configuration>
-                        <target>1.7</target>
-                        <source>1.7</source>
+                        <target>1.8</target>
+                        <source>1.8</source>
```

Comment:
Nit: should this just be `8` now?

Reply:
Yeah. We need 8 as the source to compile with Java 21.